plugins/ssh-plugin/src/ssh-plugin-backend.ts

```diff
@@ -130,8 +130,22 @@ const createKeyPair = async (sshkeyManager: SshKeyManager) => {
     }
 };
 
+const getKeys = async (sshKeyManager: SshKeyManager): Promise<cheApi.ssh.SshPair[]> => {
+    const keys: cheApi.ssh.SshPair[] = await sshKeyManager.getAll('vcs');
+    if (!keys || keys.length < 1) {
+        throw new Error('No SSH key pairs has been defined.');
+    }
+    return keys;
+};
+
 const deleteKeyPair = async (sshkeyManager: SshKeyManager) => {
-    const keys: cheApi.ssh.SshPair[] = await sshkeyManager.getAll('vcs');
+    let keys: cheApi.ssh.SshPair[];
+    try {
+        keys = await getKeys(sshkeyManager);
+    } catch (error) {
+        theia.window.showWarningMessage(error.message);
```

Comment:
should say that we're trying to delete in warning message

Reply:
done